lib/view.js

```diff
@@ -0,0 +1,56 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
```

Comment:
It does, yes. The tool could also just be run on this branch and fixed as part of this PR ;-)